### PR TITLE
[Chat][RTE] Add UI and unit tests for rich text editor components

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
# What

- Add UI and unit tests for rich text editor components
- Fixed a build issue for components tests

Please note that all tests are for RichTextInputBox, not for send box.

# Why

# How Tested

Tests

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/InputBoxButton.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ export type InputBoxButtonProps = {
   id?: string;
   ariaLabel?: string;
   tooltipContent?: string;
+  testId?: string;
```
Comment (palatter) [inline packages/react-components/src/components/InputBoxButton.tsx:22]: Did we modify tests for this component as well?

Comment (vhuseinova-msft) [reply packages/react-components/src/components/InputBoxButton.tsx:22]: It looks like we don't have any specific tests for `InputBoxButtons`

Comment (palatter) [reply packages/react-components/src/components/InputBoxButton.tsx:22]: Was just curious why we added a testId for this component?

**Review by palatter — APPROVED** [review]

**Review by palatter — APPROVED** [review]
New changes look good

**Review by Leah-Xia-Microsoft — APPROVED** [review]
LGTM

**Pull request merged**